doc/release-notes.md

```diff
@@ -10,6 +10,21 @@ Option handling
 - The `-reindex` and `-reindex-chainstate` options now imply `-rescan`
   (provided that the wallet is enabled and pruning is disabled, and unless
   `-rescan=0` is specified explicitly).
+- A new `-anchorconfirmations` argument has been added to allow the user
+  to specify the number of blocks back from the chain tip that anchors will be
+  selected from when spending notes. By default, anchors will now be selected
+  to have 3 confirmations. Values greater than 100 are not supported.
+
+RPC Interface
+-------------
+
+- The default `minconf` value for `z_sendmany` is now 10 confirmations instead
+  of 1. If `minconf` and specifies a value less than that provided for
```

Comment:
```suggestion
  of 1. If `minconf` specifies a value less than that provided for
```

Reply:
Fixed in https://github.com/zcash/zcash/pull/5934